src/index.js

```diff
@@ -34,6 +34,26 @@ export function toggleCreator(context) {
   }
 }
 
+export function handleSumbit(func){
+  return (e) => {
+    e.preventDefault();
+
+    const form = e.target;
+    let values = {};
+
+    let children = [];
+
+    children.push(form.querySelectorAll('input, select, textarea'));
+
+    for(let element of children[0]){
+
+      values = { ...values, [element.name]: element.value };
```

Comment:
Can you add a comment here about what values will look like? (Just an example or something)